modules/reactor/src/meshgenerators/PinMeshGenerator.C

```diff
@@ -70,11 +69,15 @@ PinMeshGenerator::validParams()
                         "Determines if this is the final step in the geometry construction"
                         " and extrudes the 2D geometry to 3D. If this is true then this mesh "
                         "cannot be used in further mesh building in the Reactor workflow");
+  params.addParam<bool>(
+      "homogenized", false, "Determines whether homogenized pin mesh should be generated");
+  params.addParam<bool>(
+      "use_as_assembly", false, "Determines whether pin mesh should be used as an assembly mesh");
 
   params.addParam<bool>(
       "quad_center_elements", true, "Whether the center elements are quad or triangular.");
-  params.addParamNamesToGroup("region_ids pin_type", "ID assigment");
-  params.addParamNamesToGroup("mesh_intervals ring_radii num_sectors pin_type",
+  params.addParamNamesToGroup("region_ids pin_type use_as_assembly", "ID assigment");
```

Comment:
I d move it down
while it s true it s mostly about IDs, the user isnt thinking about that when setting use_as_assembly

Reply:
Moved to ID assignment group